dev/breeze/src/airflow_breeze/commands/testing_commands.py

```diff
@@ -149,11 +268,39 @@ def tests(
         os.environ["LIST_OF_INTEGRATION_TESTS_TO_RUN"] = ' '.join(list(integration))
     if db_reset:
         os.environ["DB_RESET"] = "true"
-
-    exec_shell_params = ShellParams(verbose=verbose, dry_run=dry_run)
+    exec_shell_params = ShellParams(
+        verbose=verbose,
+        dry_run=dry_run,
+        python=python,
+        backend=backend,
+        postgres_version=postgres_version,
+        mysql_version=mysql_version,
+        mssql_version=mssql_version,
+    )
     env_variables = get_env_variables_for_docker_commands(exec_shell_params)
     perform_environment_checks(verbose=verbose)
     cmd = ['docker-compose', 'run', '--service-ports', '--rm', 'airflow']
     cmd.extend(list(extra_pytest_args))
-    result = run_command(cmd, verbose=verbose, dry_run=dry_run, env=env_variables, check=False)
+    version = (
+        mssql_version
+        if backend == "mssql"
+        else mysql_version
+        if backend == "mysql"
+        else postgres_version
+        if backend == "postgres"
+        else "none"
+    )
+    if limit_progress_output:
+        result = run_with_progress(
+            cmd=cmd,
+            env_variables=env_variables,
+            test_type=test_type,
+            python=python,
+            backend=backend,
+            version=version,
+            verbose=verbose,
+            dry_run=dry_run,
+        )
+    else:
```

Comment:
@potiuk why does `limit_progress_output` check alone need to have `python`, `backend`, `version` passed?  I could see it's used for logging alone.

Reply:
It's not the "limit progess output". It's the "test" command that missed it. The version of the tests command we had just did not have those parameters and always used defaults. Which was fine to run quick testing locally but if we are going to use "tests" command in CI, we need to test many combinations of tests (i.e. running mysql version 8 on Python 3.7). or running (Postgres 10 on python 3.10). That's why we need to specify those as parameters here (and that's why I added it to ShellParams - because they will determine which CI image (python 3.7 - 3.10) will be used for the tests.